README.md

```diff
@@ -4,27 +4,155 @@
+
 
+### 3. Running the agent
+
+To start the agent, simply run:
+```bash
+docker compose up
+```
+
+Once everything is up and running, you should see output similar to this:
+```bash
+...
+orchestrator-1   |    FastAPI   Starting production server 🚀
+orchestrator-1   |
+orchestrator-1   |              Searching for package file structure from directories with
+orchestrator-1   |              __init__.py files
+kubernetes-1     | ✅ Kubeconfig updated successfully.
+kubernetes-1     | 🚀 Starting Node.js application...
+orchestrator-1   |              Importing from /
+orchestrator-1   |
+orchestrator-1   |     module   📁 app
+orchestrator-1   |              ├── 🐍 __init__.py
+orchestrator-1   |              └── 🐍 client.py
+orchestrator-1   |
+orchestrator-1   |       code   Importing the FastAPI app object from the module with the following
+orchestrator-1   |              code:
+orchestrator-1   |
+orchestrator-1   |              from app.client import app
+orchestrator-1   |
+orchestrator-1   |        app   Using import string: app.client:app
+orchestrator-1   |
+orchestrator-1   |     server   Server started at http://0.0.0.0:80
+orchestrator-1   |     server   Documentation at http://0.0.0.0:80/docs
+orchestrator-1   |
+orchestrator-1   |              Logs:
+orchestrator-1   |
+orchestrator-1   |       INFO   Started server process [1]
+orchestrator-1   |       INFO   Waiting for application startup.
+orchestrator-1   |       INFO   Application startup complete.
+orchestrator-1   |       INFO   Uvicorn running on http://0.0.0.0:80 (Press CTRL+C to quit)
+kubernetes-1     | 2025-04-24 12:53:00 [info]: Initialising Kubernetes manager {
+kubernetes-1     |   "service": "kubernetes-server"
+kubernetes-1     | }
+kubernetes-1     | 2025-04-24 12:53:00 [info]: Kubernetes manager initialised successfully {
+kubernetes-1     |   "service": "kubernetes-server"
+kubernetes-1     | }
+kubernetes-1     | 2025-04-24 12:53:00 [info]: Starting SSE server {
+kubernetes-1     |   "service": "kubernetes-server"
+kubernetes-1     | }
+kubernetes-1     | 2025-04-24 12:53:00 [info]: mcp-kubernetes-server is listening on port 3001
+kubernetes-1     | Use the following url to connect to the server:
+kubernetes-1     | http://localhost:3001/sse {
+kubernetes-1     |   "service": "kubernetes-server"
+kubernetes-1     | }
 ```
+
+This means all the services — Slack, GitHub, the orchestrator, the prompt and the MCP servers have started successfully and are ready to handle requests.
+
+### 4. Using the agent
+
+Once the agent is up and running, you can trigger the SRE Agent by sending a request to the orchestrator service:
+
+```bash
 curl -X POST http://localhost:8003/diagnose \
   -H "accept: application/json" \
   -H "Authorization: Bearer <token>" \
   -d "text=<service>"
 ```
 
+Replace `<token>` with your dev bearer token (e.g. whatever you set in .env), and `<service>` with the name of the service you'd like the agent to investigate.
```

Comment:
```suggestion
Replace `<token>` with your dev bearer token (e.g. whatever you set in .env), and `<service>` with the name of the Kubernetes service in your target cluster you'd like the agent to investigate.
```
I think it's worth explicitly saying that we expect the service in their target cluster, not one of the services we've set up here